.github/CODEOWNERS

```diff
@@ -5,8 +5,8 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 # Note: This file is autogenerated. To make changes to the codeowner team, please update .repo-metadata.json.
 
-# @googleapis/yoshi-python @googleapis/anthos-dpe are the default owners for changes in this repo
-*     @googleapis/yoshi-python @googleapis/anthos-dpe
+# @googleapis/yoshi-python @googleapis/yoshi-python are the default owners for changes in this repo
+*     @googleapis/yoshi-python @googleapis/yoshi-python
```

Comment:
why twice?

Reply:
It's due to the file being templated. The suggestion I put should help get things fixed.

Reply:
Ohhh I see. That's OwlBot doing its thing, I think I know what's going on--